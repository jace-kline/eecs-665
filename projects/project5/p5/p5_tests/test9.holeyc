int main() {
    int x;
    int y;
    bool b;
    charptr cptr1;
    charptr cptr2;
    x = (x == y);
    if(b != x) {
        return (cptr1 == cptr2);
    }
}