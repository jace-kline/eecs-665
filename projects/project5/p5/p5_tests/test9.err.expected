FATAL [7,7]: Invalid assignment operation
FATAL [8,10]: Invalid equality operation
FATAL [9,23]: Bad return value
Type Analysis Failed
